tldextract/cache.py

```diff
@@ -38,8 +38,7 @@ def md5(*args: bytes) -> hashlib._Hash:
 
 
 def get_pkg_unique_identifier() -> str:
-    """
-    Generate an identifier unique to the python version, tldextract version, and python instance.
+    """Generate an identifier unique to the python version, tldextract version, and python instance.
```

Comment:
Do you know what generated these linebreak diffs? I'm just curious, to reduce noise in our tooling. The latest version of `black .` and `ruff format .` don't seem to care about this change either way.